webview/src/experiments/components/AddStage.tsx

```diff
@@ -22,7 +22,7 @@ export const AddStage: React.FC<AddStageProps> = ({ hasValidDvcYaml }) => (
       disabled={!hasValidDvcYaml}
     />
     {!hasValidDvcYaml && (
-      <p className={styles.error}>
```

Comment:
Similiar to the previous pr, renamed some classes to better explain what they're doing :)